paper.bib

```diff
@@ -37,6 +37,20 @@ @article{julia
   journal   = {{SIAM} Review}
 }
 
+@inproceedings{ge2018t,
```

Comment:
is there a doi for this? or is this biburl? I know the JOSS editor runs a DOI check.

Reply:
I checked and they do not have a doi, unfortunately. Just the URL, which should be "permanent" to some extent